src/Collage.py

```diff
@@ -1,13 +1,22 @@
 import tkinter as tk
 import tkinter.messagebox as messagebox
 import PIL
-from collage.CollageImage import PILCollageImage
+from src.CollageImage import PILCollageImage
+from src.CornerCreator import CornerCreator
 
 
 class Collage(tk.Canvas):
-    def __init__(self, margin, master_args, master_kwargs):
+    def __init__(
+        self,
+        margin,
+        corner_width,
+        corner_curve,
+        master_args,
+        master_kwargs
+    ):
         super().__init__(*master_args, **master_kwargs)
         self.margin = margin
+        self.corner_creator = CornerCreator(corner_width, corner_curve)
```

Comment:
Add width and curve setters here. After that, you can add tracking of curvature value, and inform user in case when this value is beyond [0, 1].